apps/gallery/js/startup.js

```diff
@@ -271,6 +271,11 @@
       // and is ready to interact with.
       window.performance.mark('visuallyLoaded');
       window.performance.mark('contentInteractive');
+
+      var headers = document.querySelectorAll('gaia-header');
+      [].forEach.call(headers, function(header) {
```

Comment:
When I suggested forEach(), I was thinking that querySelectorAll() returned a true array instead of a NodeList. If I was writing it, though, I'd use a for(;;) loop, or a for (h of headers) {} instead of having to do the [].foreach.call hack. But I'm okay with it like this if you want to leave it this way.